pom.xml

```diff
@@ -35,8 +35,19 @@
             <artifactId>log4j</artifactId>
             <version>LATEST</version>
         </dependency>
+        <dependency>
```

Comment:
@vikramvi with the latest AppiumTestDistribution extentreports are fixed and why cucumber-testng, this again part of master pom.xml